apps/web/src/app/sign-up/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { AppBar } from '@/components/common/app-bar';
+import { ReactNode } from 'react';
+import Image from 'next/image';
+
+export default function Layout({ children }: { children: ReactNode }) {
```

Comment:
- Props 의 경우 따로 type 을 지정해주는 편이 유지보수성과 확장성면에서 좋습니다.
- React 에는 `ChildrenWithProps` 라는 타입이 존재합니다.

```suggestion
type Props = PropsWithChildren<{}>

export default function Layout({ children }: Props) {
```